NurVo_Frontend/src/components/LibraryHeader.tsx

```diff
@@ -0,0 +1,151 @@
+import { useState } from 'react';
+import { View, Text, TouchableOpacity, StyleSheet, Image } from 'react-native';
+import { useNavigation } from '@react-navigation/native';
+import Ionicons from 'react-native-vector-icons/Ionicons';
+
+import ImformationAlert from './ImformationAlert';
+import Colors from '../utilities/Color';
+
+interface LibraryHeaderProps {
+  title: string,
+  image?: {
+    src: string,
+    alt: string,
+  }[];
+  isBack?: boolean,
+  isDelete?: boolean,
+}
+
+const LibraryHeader = ({ title, image, isBack, isDelete }: LibraryHeaderProps) => {
+  const navigation = useNavigation(); // 네비게이션 객체 받아오기
+  const isback = isBack || false;
+  const isdelete = isDelete || false;
+  const [openItemBox, setOpenItemBox] = useState(false);
+  const [isDeleteBar, setIsDeleteBar] = useState(false);
+
+  const openitem = () => {
+    setOpenItemBox((prev) => !prev);
+  }
+
+  const openDelete = () => {
+    navigation.navigate('Library', { data: true });
+    setOpenItemBox(false);
+    setIsDeleteBar(true);
+  }
+
+  const closeDelete = () => {
+    navigation.navigate('Library', { data: false });
+    setIsDeleteBar(false);
+  }
+
+  return (
+    <View>
+      {!isDeleteBar ?
+        <View style={navigationBarStyles.Container}>
+          <View style={navigationBarStyles.headerContainer}>
+            {isback ?
+              <Ionicons
+                name="arrow-back"
+                size={23.5}
+                color="black"
+                style={{ marginHorizontal: 18, }}
+                onPress={() => navigation.goBack()}
+              />
+              : null
+            }
+            <Text style={navigationBarStyles.headerTitle}>{title}</Text>
+            <TouchableOpacity>
+              {image ? <ImformationAlert image={image} /> : null}
+            </TouchableOpacity>
+          </View>
+
+          {isdelete ?
+            <Ionicons
+              name="ellipsis-vertical"
+              size={30}
```

Comment:
```suggestion
              size={20}
```
다른 부분에 비해 아이콘이 너무 커서 이질감이 느껴지는 것 같아 크기를 줄이는 것이 좋을 것 같습니다.